.github/workflows/main.yml

```diff
@@ -0,0 +1,40 @@
+name: build
+
+on: [ push, pull_request ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install packages
+      run: |
+        sudo apt-get -y install pandoc
+      if: matrix.python-version == 3.6
```

Comment:
how should I read this syntax format?  Install `pandoc` only for python 3.6?

Reply:
Is the indentation wrong?  Compared to other `if` below?

Reply:
Exactly. This `if` is for GitHub actions, whereas the `if`'s below are for the shell.